File: workout/beecrowd/code/1012.java
import java.io.IOException;
import java.util.Scanner;
/**
 * IMPORTANT: 
 *      O nome da classe deve ser "Main" para que a sua solução execute
 *      Class name must be "Main" for your solution to execute
 *      El nombre de la clase debe ser "Main" para que su solución ejecutar
 */
public class Main {
 
    public static void main(String[] args) throws IOException {
            Scanner leitor = new Scanner(System.in);
            
            double a = leitor.nextDouble();
            double b = leitor.nextDouble();
            double c = leitor.nextDouble();
            
            double triangle =  (a * c) / 2;
            double circle = 3.14159 * c * c;
            double trapezium = (a + b) * c / 2;
            double square = b * b;
            double rectangle = a * b;
            
            System.out.printf(
                "TRIANGULO: %.3f%n" +
                "CIRCULO: %.3f%n" +
                "TRAPEZIO: %.3f%n" +
                "QUADRADO: %.3f%n" +
                "RETANGULO: %.3f%n",
                triangle,
                circle,
                trapezium,
                square,
                rectangle
            );
        }
} 

File: workout/beecrowd/code/1012.cs
using System;

public class URI{
    static void Main(string[] args){
        string entrada = Console.ReadLine();
        string[] numeros = entrada.Split(' ');

        double a = double.Parse(numeros[0]);
        double b = double.Parse(numeros[1]);
        double c = double.Parse(numeros[2]);

        double triangle =  (a * c) / 2.0;
        double circle = 3.14159 * c * c;
        double trapezium = (a + b) * c / 2.0;
        double square = b * b;
        double rectangle = a * b;

        Console.WriteLine($"TRIANGULO: {triangle:F3}");
        Console.WriteLine($"CIRCULO: {circle:F3}");
        Console.WriteLine($"TRAPEZIO: {trapezium:F3}");
        Console.WriteLine($"QUADRADO: {square:F3}");
        Console.WriteLine($"RETANGULO: {rectangle:F3}");

    }
}